app/pages/sections/order-items/catalogue-solutions/delete/template.njk

```diff
@@ -14,9 +14,9 @@
     <div class="nhsuk-grid-row">
       <div class="nhsuk-grid-column-two-thirds">
         <h1 data-test-id="delete-catalogue-page-title" class="nhsuk-u-margin-top-7 nhsuk-u-margin-bottom-3">{{ title }}</h1>
-        <h2 data-test-id="delete-catalogue-page-description" class="nhsuk-u-font-size-24 nhsuk-u-margin-bottom-6">{{ description }}</h2>
-        <h3 data-test-id="catalogue-description-title" class="nhsuk-u-margin-bottom-3">{{ orderDescriptionTitle }}</h3>
-        <h4 data-test-id="catalogue-description" class="nhsuk-u-font-size-19 nhsuk-u-font-weight-normal">{{ orderDescription }}</h4>
+        <p data-test-id="delete-catalogue-page-description" class="nhsuk-heading-l nhsuk-u-font-size-24 nhsuk-u-margin-bottom-6">{{ description }}</p>
+        <h2 data-test-id="catalogue-description-title" class="nhsuk-u-margin-bottom-3">{{ orderDescriptionTitle }}</h2>
```

Comment:
Is any class missing here? As converting h3 to h2 will increase font size, not sure if this is intended.

Reply:
It will be different but it should look good.